src/main/java/io/jenkins/plugins/services/impl/HttpClientJiraIssues.java

```diff
@@ -44,21 +45,34 @@ public String getHttpContent(String url, List<Header> headers) {
     return super.getHttpContent(url, headers);
   }
 
+  @Override
+  protected boolean isValidStatusCode(int statusCode) {
+    return statusCode == HttpStatus.SC_OK || statusCode == HttpStatus.SC_BAD_REQUEST;
+  }
+
   public JiraIssues getIssues(String pluginName) throws IOException {
     return getIssues(pluginName, 0);
   }
 
   public JiraIssues getIssues(String pluginName, int startAt) throws IOException {
     int maxResults = 100;
+    String component = pluginName.replaceAll("-plugin$", "") + "-plugin";
     JiraIssues jiraIssues = new JiraIssues();
 
-    String query = URLEncoder.encode("project=JENKINS AND status in (Open, \"In Progress\", Reopened) AND component=" + pluginName.replaceAll("-plugin$", "") + "-plugin", "UTF-8");
+    String query = URLEncoder.encode("project=JENKINS AND status in (Open, \"In Progress\", Reopened) AND component=" + component, "UTF-8");
     String jsonInput = getHttpContent("/rest/api/2/search?startAt=" + startAt + "&maxResults=" + maxResults + "&jql=" + query, Collections.emptyList());
     if (Strings.isNullOrEmpty(jsonInput)) {
       throw new IOException("Empty return value");
     }
 
     JSONObject obj = new JSONObject(jsonInput);
+    if (obj.has("errorMessages")) {
+      if (obj.getJSONArray("errorMessages").join("|").contains("\"The value '" + component + "' does not exist for the field 'component'.\"")) {
+        logger.debug("JSON Response with error: " + jsonInput);
```

Comment:
Should this be logged at info and contain the plugin name?

Reply:
actually I don't think it should be logged at all now that you mention it. 

We can't really do anything about it unless someone can go rename components.

Reply:
i was wondering if it should be logged so that we know about the issue and can go look at fixing it, otherwise we're just getting this error without knowing about it

Reply:
Thats exactly my question.
If we can do something about it, sentry is better
if we can't do anything about it, logging is useless.

Reply:
@slide pinging you here cause it makes sense to keep the conversation here instead of splitting on irc.

Reply:
Are we concerned about breaking any queries or anything
if you search for `{job="plugin-site/plugin-site-backend"} |= "Unable"` in loki, you can see the failing queries.

so for example, `sonar-plugin` doesn't exist but `sonar` does

Reply:
I doubt it